SPECS/systemd/systemd.spec

```diff
@@ -39,9 +39,8 @@ Requires:       libgcrypt
 Requires:       lz4
 Requires:       pam
 Requires:       xz
-
+Requires(pre):  audit-libs
```

Comment:
Btw. how do we know `systemd` requires `audit-libs` to run now? The installation errors I've seen so far didn't complain about any of the libraries coming from `audit-libs`. If we really need it, then we're still stuck, because `audit` clearly requires `systemd` during its `%post` section, so `systemd` needs to up-and-running before we install `audit`, which drags in `audit-libs`.

Reply:
The following error was seen when installing systemd:
WARN[0014] systemctl: error while loading shared libraries: libaudit.so.1: cannot open shared object file: No such file or directory